igluctl/0.5.0/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM snowplow-docker-registry.bintray.io/snowplow/base-alpine:0.2.0
+LABEL maintainer="Snowplow Analytics Ltd. <support@snowplowanalytics.com>"
+
+# Bin path
+ENV BIN_PATH="/usr/local/bin"
```

Comment:
we have a `SNOWPLOW_BIN_PATH` where we usually install stuff

Reply:
@BenFradet that's within the path where everything is mounted so we need to:
1) Change the volume location, but I do think '/snowplow'  is easy to remember to mount
2) Keep as is

This is also the reason I went with custom image in the first place as it doesn't need anything from the base image.

Reply:
@BenFradet 
OK so if I leave as is you can run:
`docker run -v ${PWD}:/snowplow igluctl:0.5.0 lint "schemas/"`

If I use `SNOWPLOW_BIN_PATH` which is actually `/snowplow/bin`, then I can't mount /snowplow as above because that will make the binarly unavailable in the container. My original commit had 'igluctl' as the directory like so: 
`docker run -v ${PWD}:/igluctl igluctl:0.5.0 lint "schemas/"`

Mounting the schema registry into a config folder  doesn't make sense to me. It would look like so:
`docker run -v ${PWD}:/snowplow/config igluctl:0.5.0 lint "schemas/"`

Based on that I think we either change the mount to `/igluctl` or just leave as is.